### PR TITLE
[WIP] Add license file

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,10 @@
+By accessing NServiceBus code here, you are agreeing to the following licensing terms.
```
Comment (HEskandari) [inline LICENSE]: Should this be renamed to ServiceInsight?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+If you do not agree to these terms, do not access the NServiceBus code.
+
+Your license to the NServiceBus source and/or binaries is governed by the Reciprocal Public License 1.5 (RPL1.5) license as described here:
```
Comment (HEskandari) [inline LICENSE]: Same as above.

File: LICENSE (hunk 1 above)
Comment (johnsimons) [inline LICENSE]: Not really, we are registered as NServiceBus

Comment (distantcam) [inline LICENSE]: Registered?

Comment (johnsimons) [inline LICENSE]: Yes officially we are NServiceBus doing business as Particular Software

**Comment by johnsimons** [comment]
Trying to clarify, see https://github.com/Particular/PlatformDevelopment/issues/983#issuecomment-257498951

File: LICENSE (hunk 1 above)
Comment (HEskandari) [inline LICENSE]: This makes sense when read off NServiceBus repo, but not ServiceInsight and / or ServicePulse. If 'NServiceBus' referred to here means the company, could it be clarified? I mean, this is a license that people would need to read and understand clearly so why leave it vague?

Comment (johnsimons) [inline LICENSE]: https://github.com/Particular/PlatformDevelopment/issues/983#issuecomment-257498951

**Comment by adamralph** [comment]
Added to 1.5.9 milestone.

**Pull request merged**